src/components/command-bar/commands/search/components/tutorials-tab-contents/types.ts

```diff
@@ -11,15 +16,25 @@ interface TutorialsTabContentsProps {
 
 type TutorialHitObject = Hit<{
 	description: string
-	name: string
-}> & {
-	defaultContext: {
-		slug: string
+	page_title: string
+	name: string // TODO: remove this once page_title is fully seeded
+	headings: string[]
+}> &
+	Pick<
+		TutorialLite,
+		| 'defaultContext'
+		| 'id'
+		| 'name'
+		| 'slug'
+		| 'description'
+		| 'readTime'
+		| 'edition'
+	> & {
+		hasVideo: boolean
+		isInteractive: boolean
+		products: ProductOption[]
+		level: CollectionLevelOption
```

Comment:
I hate this but I was having trouble with the `BaseHit` requirements if I tried to make this into an interface that extends these types.